source/megengine.js

```diff
@@ -375,20 +516,70 @@ megengine.Argument = class {
     get initializer() {
         return this._initializer;
     }
+
+    get quantization() {
+        if (this._scale !== undefined && this._zeroPoint !== undefined) {
+            return this._scale.toString() + ' * ' + (this._zeroPoint == 0 ? 'q' : '(q - ' + this._zeroPoint.toString() + ')');
+        }
+        return undefined;
+    }
 };
 
 megengine.Node = class {
 
-    constructor(metadata, item) {
-        this._name = item.name || '';
-        this._type = Object.assign({}, metadata.type(item.type));
-        if (this._type.name.length > 4 && this._type.name.startsWith('__') && this._type.name.endsWith('__')) {
-            this._type.name = this._type.name.substring(2, this._type.name.length - 2);
+    constructor(metadata, item, gather_opr_tensor) {
+        switch (megengine.modelType) {
+            case 'megengine.pickle': {
+                this._name = item.name || '';
+                this._type = Object.assign({}, metadata.type(item.type));
+                if (this._type.name.length > 4 && this._type.name.startsWith('__') && this._type.name.endsWith('__')) {
+                    this._type.name = this._type.name.substring(2, this._type.name.length - 2);
+                }
+                this._inputs = [];
+                this._outputs = [];
+                this._chain = [];
+                this._attributes = [];
+                break;
+            }
+            case 'megengine.flatbuffers': {
+                this._name = '';
+
+                this._type = Object.assign({}, metadata.type(item.type));
+                this._type.name = this._type.name.replace(/V(\d+)$/, '');
+                this._type.category = this._type.category ? this._type.category : 'Layer';
```

Comment:
The coloring for many layers is different from other formats. Use the same categories as other formats consistently. For example, `Layer` is not the default and should only be used for Convolution, Dense and LSTM layers. `Reshape` is category `Shape`.

Reply:
I will fix this bug next week. Have a nice weekend!

Reply:
Add `metadata` to `./tools/megengine` and `./Makefile` to invoke the script as part of `make update`.

Reply:
done